### ISSUE TITLE
For `text area` component, with `Full Keyboard Access` the naivgation traps the user and he/she cannot esapce with TAB

### ISSUE THREAD

**Issue by pylapp** [Orange-OpenSource/ouds-ios#1645]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 14 Pro

### Device OS version

iOS 26.5

### Is Liquid Glass disabled

- [ ] Yes, Liquid Glass is disabled (or OS version is older than 26)
- [x] No, Liquid Glass is enabled (or OS version is 26 or newer)

### OUDS iOS library version

3.0.0-beta.7

### Xcode version

Xcode 26.5 Build version 17F42

### Swift version

swift-driver version: 1.148.6 Apple Swift version 6.3.2 (swiftlang-6.3.2.1.108 clang-2100.1.1.101) Target: arm64-apple-macosx26.0

### App orientation

Portrait

### App language

French

### Device language

French

### Describe the issue

As a user with _Full Keyboard Access (FKA),
When I navigate to the `text area` OUDS component,
I am stuck in the component and the TAB key does not make me leave the field,
So I need to ESC and I loose what I wrote_

In screenshot below, in pink rectangle the system tooltip, in blue dashed rectanbge the trapping text area.

<img height="500" alt="Image from another sandbox app" src="https://github.com/user-attachments/assets/89d3161e-f173-46f9-a3a1-2c2306f42528" />

### Expected Behavior

TAB or key combinations can let me escape the _text area_

### Accessibility settings

Full Keyboard Access

**Comment by pylapp** [comment]
> [!TIP]
> There are shortcuts to escape
> like CTRL + TAB

<img height="500" alt="Image" src="https://github.com/user-attachments/assets/de5c9a0d-0583-45e6-be27-da7b38d04443" />

**Comment by pylapp** [comment]
> [!NOTE]
> **On hold** until @pya35 checks if the use of shortcuts is enough

**Comment by pya35** [comment]
After checking W3C experts opinion, `CTRL+TAB `is accepted on iOS, as a way of navigating with keyboard if arrows or `TAB `keys don't work.
Issue closed
